droidlet/interpreter/robot/default_behaviors.py

```diff
@@ -101,3 +106,28 @@ def explore(agent):
     x,y,t = agent.mover.get_base_pos()
     goal = get_distant_goal(x,y,t)
     start_explore(agent, goal)
+
+def get_task_data(agent):
+    with open(agent.opts.reexplore_json, 'r') as f:
```

Comment:
try catch this with block, and then return `None` in the catch, maybe?

Reply:
done.